src/parser/jobs/sch/modules/DoTs.js

```diff
@@ -45,17 +52,16 @@ export default class DoTs extends CoreDoTs {
 	}
 
 	addClippingSuggestions(clip) {
-
-		const maxClip = this.getClippingAmount(STATUSES.BIOLYSIS.id)
+		const clipPerMinute = this.getClippingAmount(STATUSES.BIOLYSIS.id)
 		this.suggestions.add(new TieredSuggestion({
 			icon: ACTIONS.BIOLYSIS.icon,
 			content: <Trans id="sch.dots.suggestions.clipping.content">
 				Avoid refreshing Biolysis significantly before its expiration, except when at the end of the fight. Unnecessary refreshes use up your mana more than necessary, and may cause you to go out of mana.
 			</Trans>,
-			tiers: CLIPPING_SEVERITY,
-			value: maxClip,
+			tiers: SEVERITIES.CLIPPING,
+			value: clipPerMinute,
 			why: <Trans id="sch.dots.suggestions.clipping.why">
-				{this.parser.formatDuration(clip[STATUSES.BIOLYSIS.id])} of <StatusLink {...STATUSES.BIOLYSIS}/> lost to early refreshes.
+				An average of {this.parser.formatDuration(this.getClippingAmount(STATUSES.BIOLYSIS.id) * 1000)} of <StatusLink {...STATUSES.BIOLYSIS}/> clipped every minute, for a total of {this.parser.formatDuration(clip[STATUSES.BIOLYSIS.id])} lost to early refreshes.
```

Comment:
No need to call `this.getClippingAmount()` again here; you should just reuse the `clipPerMinute` you defined earlier and used for the `value` field.

Reply:
Alright, got the requested fix in – thanks for the catch!